### PR TITLE
Remove dynamic PGO option and make it default

### PR DESCRIPTION
**Pull request by VasilisThePikachu**
As far as I know, no one has experienced an issue with it. May as well remove some clutter. And make sure its on by default

### CODE REVIEW

**Comment by deathride58** [comment]
The setting is somewhat useful; dynamic PGO does add a bit of startup time to the client, due to how large the assembly is. Though making it default would probably be a good idea.

**Comment by VasilisThePikachu** [comment]
> The setting is somewhat useful; dynamic PGO does add a bit of startup time to the client, due to how large the assembly is. Though making it default would probably be a good idea.

I would take client performance over a little bit extra startup time

**Comment by deathride58** [comment]
The extra startup time is not insignificant on lower-end hardware

**Comment by VasilisThePikachu** [comment]
How much longer is it? And how low end are we talking?

**Comment by PJB3005** [comment]
> The extra startup time is not insignificant on lower-end hardware

Do you have numbers on this?

**Comment by deathride58** [comment]
We do not, admittedly! Plus it's been quite a while since we last poked at it ourself (and coincidentally it's been quite a while since we had a computer where it actually mattered). It's something that should probably be benchmarked proper.

**Comment by PJB3005** [comment]
We could turn the checkmark into something that disables Full PGO instead, and clearly mark it as such.

**Comment by VasilisThePikachu** [comment]
So this is on decently good hardware. But having the option turned on or off does not impact startup time at all.

**Comment by VasilisThePikachu** [comment]
Some more findings: on a MacBook air m1

With pgo enabled: 30 second to get from launcher to game
With pgo off: 25 seconds

File: SS14.Launcher/Models/Connector.cs [hunk 1]
```diff
@@ -462,13 +462,10 @@ private async Task<(ServerInfo, Uri, Uri)> GetServerInfoAsync(string address, Ca
...
-        if (_cfg.GetCVar(CVars.DynamicPgo))
-        {
-            Log.Debug("Dynamic PGO is enabled.");
-            EnvVar("DOTNET_TieredPGO", "1");
-            EnvVar("DOTNET_TC_QuickJitForLoops", "1");
-            EnvVar("DOTNET_ReadyToRun", "0");
-        }
+        // Performance tweaks
+        EnvVar("DOTNET_TieredPGO", "1");
+        EnvVar("DOTNET_TC_QuickJitForLoops", "1");
```
Comment (PJB3005) [inline SS14.Launcher/Models/Connector.cs:467]: `DOTNET_TC_QuickJitForLoops` is default since .NET 7, remove the set for it.

**Review by PJB3005 — CHANGES_REQUESTED** [review]

**Pull request merged**